src/main/java/org/ecocean/servlet/importer/StandardImport.java

```diff
@@ -2657,29 +2657,16 @@ private void sendforACMID(ImportTask itask, Shepherd myShepherd, String context)
                 ArrayList<MediaAsset> assets = new ArrayList<MediaAsset>();
                 for (Encounter enc : allEncs) {
                     count++;
-
+                    itask.setStatus("Registering image assets for " + count + "/" + numEncs +
+                            " encounters.");
                     ArrayList<MediaAsset> theseAssets = enc.getMedia();
                     for (MediaAsset assy : theseAssets) {
                         if (!assy.hasAcmId()) {
-                            assets.add(assy);
+                        	IAGateway.addToAcmIdQueue(context, Integer.toString(assy.getId()));
```

Comment:
compiling is failing here as second argument to `addToAcmIdQueue()` appears to be a _String_ here.

Reply:
Updated method reference to new String default (old was Int default)